tests_integration/__tests__/trim.js

```diff
@@ -0,0 +1,62 @@
+"use strict";

```

Comment:
Could you rename this file to `doc-trim.js`? so that it'd be clearer for what's this file used for.